monitoring/bin/deploy_monitoring_cluster.sh

```diff
@@ -229,6 +220,24 @@ else
   log_debug "PrometheusRule $MON_NS/v4m-kubernetes-apps does not exist"
 fi
 
+# Elasticsearch Datasource for Grafana
+# Moved down to make sure Grafana pods exist
```

Comment:
This looks like a message to code reviewers rather a comment for future developers.  It's only meaningful if you know what was here before.  There's no reason to remove it or reword it (necessarily) but remember who the audience for the comments are.